test_login.py

```diff
@@ -13,6 +12,7 @@ def test_login(base_url, selenium, fxa_account):
     assert page.logged_in
 
 
+@pytest.mark.skip
```

Comment:
Can we add a reason here, just so we remember why we skipped. https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions